.github/workflows/bump-version.yml

```diff
@@ -29,7 +33,7 @@ on:
       - pyproject.toml
       - tests/**.py
       - tox.ini
-      - xsdba/__init__.py
+      - src/xsdba/__init__.py
```

Comment:
I'm confused about the change, since it's a whole new repo. Is this something about CookieCutter previously not respecting this structure with `src/...`?

Reply:
The first commit was a generic Github template, and now you're applying your own CookieCutter template?

Reply:
There are a lot of moving parts and sometimes I forget to add certain files. The `src` structure is relatively, and I'm still moving the template over to it, so it seems.

I'll probably be pushing one more change (the issue about numpydoc that I raised is fixed). I'll merge soon after!